src/drawing/DrawingTarget.scss

```diff
@@ -1,7 +1,28 @@
+$ba-DrawingTarget-transitionDelay: 25ms;
+$ba-DrawingTarget-transitionDuration: 200ms;
+
 .ba-DrawingTarget {
     outline: none;
 
+    path {
```

Comment:
Can we add appropriate CSS classes to our SVG elements to avoid referring to them via tag name?